bedrock/firefox/templates/firefox/dashboard-base.html

```diff
@@ -7,7 +7,7 @@
 {% block page_title_prefix %}{% endblock %}
 {% block page_title %}{% endblock %}
 
-{% block site_css %}
+{% block page_css %}
   {% stylesheet 'firefox-dashboard' %}
```

Comment:
@jpetto Thanks for explanation. I've also seen 

{% block extrahead %}
  {% stylesheet 'foo' %}
{% endblock %}

Does this do the same? Just asking to understand. Either way I've updated to the changes you have noted above. So now child templates extending from dashboard-base.html will have a page_css block instead.